### ISSUE TITLE
Support more data formats/integration with other PKMs

### ISSUE THREAD

**Issue by zimengzhou1** [reorproject/reor#276]
What are our thoughts on supporting other document types other than markdown, for example PDF or plaintext? Also it would be nice if users could directly use other note taking apps like notion as a source of their data, it would provide a lower barrier to entry to using reor.

**Comment by samlhuillier** [comment]
Absolutely! This is something that would be great to add, particularly supporting plain text & supporting pdfs. Would you be keen to add this?

**Comment by zimengzhou1** [comment]
Sure, I'll have a crack at it!

**Comment by BradKML** [comment]
Seconding the idea of data formats, but PDFs (and ebooks to a lesser extent) are a complete beast of their own, worth thinking about clearly.
